### PR TITLE
HHH-13986 Add test

### PR DESCRIPTION
**Pull request by dreab8**
https://hibernate.atlassian.net/browse/HHH-13986

### CODE REVIEW

File: hibernate-core/src/test/java/org/hibernate/id/SequenceStyleGeneratorBehavesLikeSequeceHiloGeneratorWitZeroIncrementSizeTest.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
```
Comment (Sanne) [inline hibernate-core/src/test/java/org/hibernate/id/SequenceStyleGeneratorBehavesLikeSequeceHiloGeneratorWitZeroIncrementSizeTest.java:6]: why do you have the duplicated license header?

You might want to check your templates, I had found other classes with the similar thing in the past (and have cleaned them up)

Comment (dreab8) [reply hibernate-core/src/test/java/org/hibernate/id/SequenceStyleGeneratorBehavesLikeSequeceHiloGeneratorWitZeroIncrementSizeTest.java:6]: I fixed the template, it seemed the line `#parse("File Licence.java")` is not anymore necessary

Comment (dreab8) [reply hibernate-core/src/test/java/org/hibernate/id/SequenceStyleGeneratorBehavesLikeSequeceHiloGeneratorWitZeroIncrementSizeTest.java]: removed also the duplicate header

**Comment by Sanne** [comment]
merged! thanks

**Pull request closed without merging**